README.md

```diff
@@ -31,11 +31,8 @@ sudo apt-get install libssl-dev libuv1-dev cmake make curl g++
 Clone the Gecko repository:
 
 ```sh
-cd $GOPATH
-mkdir -p src/github.com/ava-labs
-cd src/github.com/ava-labs
-git clone https://github.com/ava-labs/gecko.git
-cd gecko
+go get github.com/ava-labs/gecko
```

Comment:
As mentioned by @swdee this should be `go get -d github.com/ava-labs/gecko/...` to avoid the warning about no go files living at the top level.